src/core/locator/qgslocatorfilter.h

```diff
@@ -57,14 +57,14 @@ class CORE_EXPORT QgsLocatorResult
      *
      * \since QGIS 3.18
      */
-    QVariant getUserData() const;
+    QVariant getUserData() const SIP_PYNAME( userData );
```

Comment:
```suggestion
    QVariant userData() const SIP_PYNAME( _userData );
```

Reply:
done, and related .cpp changes to take into account the method renaming